### ISSUE TITLE
Is this updated for Diablo Season 4?

### ISSUE THREAD

**Issue by discarded-odairros** [bytemind-de/d4-tools#1]
I'd like to know which season these calculations are updated for (I'm not sure if it's necessary to update, if anything changed from **S3** to **S4**). Also, I'd like to suggest a **multi-language** option. Here in Brazil, where I'm from, we'd like a tool like this to assist with calculations, but since it's in English, it's a bit challenging. 

There has been discussion in the community about having a damage calculator with explanations about the calculations, and I found this project.

**Comment by fquirin** [comment]
Hi @OdairRosario,

technically it hasn't been updated yet for S4, but the only difference is the armor cap of ~9200 for the defense. Some of the predefined modifiers don't exist anymore, but thats just a convenience issue. I'll try to update this in the next days.

About multi-language support. I'd like to add this, but there is no feature implemented yet to replace names and tool tips with text from a language specific source file. I'll keep it in mind though and try to implement support step by step whenever I do an update. For some parts it will be easy (like the predefined affixes) and for some it will be a bit more work (tool tips, menu entries etc.).

**Comment by TalkingToWalls** [comment]
Any chance you have a full data base of everything that I could use AI with? I want it to do the calculations for me and find best stats and gear ect.

**Comment by fquirin** [comment]
> Any chance you have a full data base of everything that I could use AI with? I want it to do the calculations for me and find best stats and gear ect. 

All I have is in this repository ^^. But there isn't really much "data" except some names of aspects etc. Blizzard changes the values all the time and in addition, players can have different rolls.

Not sure if an "AI" would be able to help you with this anyway, certainly not something like ChatGPT. What you'd need is a program that can calculate all possible results with the correct formulas and then maximize the result ;)

**Comment by jabberwock** [comment]
Yes it can. https://github.com/jabberwock/claude-ipc/ first screen shot.

**Comment by fquirin** [comment]
> Yes it can. ... first screen shot.

Not sure what I'm supposed to see there, but you are welcome to explain how you think an AI agent would solve the task.

**Comment by jabberwock** [comment]
Or maybe I just did exactly what the other user was asking for without talking them down. Maybe that data was taken from both the CASC fs and maxroll's JSON. And maybe I used AI for all of it. Maybe that data is sitting in https://github.com/jabberwock/d4builder/ and being worked on.
